.github/workflows/test.yml

```diff
@@ -7,6 +7,9 @@ on:
       - "dependabot/**"
   pull_request:
 
+env:
+  FORCE_COLOR: 2
```

Comment:
This doesn't gain us a lot and arguably doesn't belong in this PR, so I can either split it or drop it. Only the `npm ci` step currently benefits.